files/files.go

```diff
@@ -123,3 +124,24 @@ func Hash(file string) (s string) {
 	}
 	return
 }
+
+// Slower than Open as it must cycle through every element in map. Open all files that match glob.
+func OpenGlob(name string) ([]io.ReadCloser, error) {
+	readers := make([]io.ReadCloser, 0)
+	for file := range staticFiles {
+		matches, err := filepath.Match(name, file)
```

Comment:
path.Match is more appropriate here, as filepath behaves differently on windows, while the paths in staticFiles is unix-style